api/resources_portal/management/commands/material_additional_metadata_key_change.py

```diff
@@ -0,0 +1,183 @@
+from django.core.management.base import BaseCommand, CommandError
+from django.utils import timezone
+
+from resources_portal.config.logging import get_and_configure_logger
+from resources_portal.models import Material
+
+logger = get_and_configure_logger(__name__)
+
+
+def validate_category(category):
+    return len([choice for choice in Material.CATEGORY_CHOICES if category in choice]) > 0
+
+
+def paginate_queryset(query_set, function, pagination):
+    current_page = 0
```

Comment:
Could we change `current_page` to `current_index`? `current_page` makes me think it's whether we're on the first page or the second page, etc, with each one have `pagination` items in it.

Reply:
Oh and maybe `pagination` -> `page_size`.

Reply:
Good ideas, much clearer. Done!